### PR TITLE
Delivers #108022488] gsa18f roles

### PR DESCRIPTION
**Pull request by pkarman**
Story: https://www.pivotaltracker.com/story/show/108022488

No functional change, just switches from environment variables to roles, similar to what we did with NCR a month+ ago.

### CODE REVIEW

File: db/seeds.rb [hunk 1]
```diff
@@ -7,3 +7,4 @@
 # it should be safe to run multiple times in the same environment, and to remove
 # the relevant ENV vars after it has run at least once.
 RolesConversion.new.ncr_budget_approvers
+RolesConversion.new.gsa18f_approvers
```
Comment (jessieay) [inline db/seeds.rb]: I am afraid that adding this is going to make our slow test suite even slower -- did working on this PR give you any good ideas on how we could move away from needing to run seeds before each test? I haven't put much thought into it recently but I am planning on doing so soon!

Comment (pkarman) [inline db/seeds.rb]: @jessieay 2cbbf07 should improve overall test suite runtime.

Comment (jessieay) [inline db/seeds.rb]: ok, that's a good start! I am still *really* hoping to find a way to not run seeds, but baby steps :baby: :chart_with_upwards_trend:

**Comment by jessieay** [comment]
Looks :bomb: -- mergin!

**Pull request merged**